lib/src/status.rs

```diff
@@ -305,18 +308,106 @@ pub(crate) async fn status(opts: super::cli::StatusOpts) -> Result<()> {
     let legacy_opt = if opts.json {
         OutputFormat::Json
     } else {
-        OutputFormat::Yaml
+        if  std::io::stdout().is_terminal() {
+            OutputFormat::HumanReadable
+        } else {
+            OutputFormat::Yaml
+        }
     };
     let format = opts.format.unwrap_or(legacy_opt);
     match format {
         OutputFormat::Json => serde_json::to_writer(&mut out, &host).map_err(anyhow::Error::new),
         OutputFormat::Yaml => serde_yaml::to_writer(&mut out, &host).map_err(anyhow::Error::new),
+        OutputFormat::HumanReadable => human_readable_output(&mut out, &host),  
     }
     .context("Writing to stdout")?;
 
     Ok(())
 }
 
+#[derive(Default, Iterable)]
+struct ImageOpts {
+    version: String,
+    transport: String,
+    digest: String,
+}
+
+fn human_readable_output(mut out: impl Write, host: &Host) -> Result<()> {
+    for (status_string, status) in [
+        ("staged", &host.status.staged),
+        ("booted", &host.status.booted),
+        ("rollback", &host.status.rollback),
+    ] {
+        if let Some(host_status) = status {
+            let mut image_opts: ImageOpts = ImageOpts::default();
+            if let Some(image) = &host_status.image {
+                out.write_all(format!("Current {:?} image: {:?}\n", status_string, image.image.image).as_bytes())?;
+
+                if let Some(version) = &image.version {
+                    image_opts.version = version.to_owned();
+                } else {
+                    image_opts.version = ("No image version defined").to_owned();
+                }
+
+                image_opts.transport = image.image.transport.clone();
+                image_opts.digest = image.image_digest.clone();
+
+                for (field_name, field_value) in image_opts.iter() {
+                    if let Some(field_print) = field_value.downcast_ref::<String>() {
+                        out.write_all(format!("     Image {}: {:?}\n", field_name, field_print).as_bytes())?;
+                    }
+                }
+
+            } else {
+                out.write_all(format!("No image defined").as_bytes())?;
+            }
+        } else {
+            out.write_all(format!("No {:?} image present \n", status_string).as_bytes())?;
+        }
+    }
+    Ok(())
+}
+
+#[test]
+fn test_human_readable() {
+    // Tests Staged and Booted, null Rollback
+    let mut SPEC_FIXTURE: &str = include_str!("fixtures/spec.yaml");
+    let mut host: Host = serde_yaml::from_str(SPEC_FIXTURE).unwrap();
+    let mut w = Vec::new();
+    human_readable_output(&mut w, &host).unwrap();
+    let w = String::from_utf8(w).unwrap();
+    dbg!(&w);
+    assert!(w.contains("quay.io/example/someimage:latest"));
```

Comment:
I think it'd be really useful here to compare literal strings inline in the tests because that'd make it way easier for reviewers to see your proposed output without building your PR. Comparing the strings is nice with `similar_asserts::assert_eq!` which we use in a few tests already in this project.